### PR TITLE
Update Visualization v3.0.0

### LINKED ISSUE #891: Review & update Tutorial: Visualizing Node Metrics to v3 compatible
https://substrate.dev/docs/en/tutorials/visualize-node-metrics/

### PR DESCRIPTION
**Pull request by NukeManDan**
As the tutorial here actually only depends on Prometheus metrics, and AFAIK these have not changed, there is no updated needed. I updated text a bit to make this a bit more clear.

Closes #891

### CODE REVIEW

**Comment by NukeManDan** [comment]
@jimmychu0807 this should be good to go once #911 lands

**Review by jimmychu0807 — APPROVED** [review]
Updated to resolve conflict. Thanks @NukeManDan

**Pull request merged**
